### PR TITLE
Added dropdowm for event suggestions in volunteer/report

### LINKED ISSUE #382: Show event suggestion as dropdown
volunteer/report/

### PR DESCRIPTION
**Pull request by kamsuri**
Fixes #382 
Added dropdown for event suggestions in volunteer/report.
screenshot from 2017-10-24 16-33-27

### CODE REVIEW

File: vms/volunteer/views.py [hunk 1]
```diff
@@ -98,7 +100,7 @@ def form_valid(self, form):
             else:
                 return render(self.request, 'volunteer/edit.html',
                               {'form': form, 'organization_list': organization_list, 'volunteer': volunteer,
-                               'resume_invalid': True,})
+                               'resume_invalid': True})
```
Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: 👋 is this autopep8, also above new lines?

File: vms/volunteer/views.py [hunk 2]
```diff
@@ -152,14 +155,20 @@ def post(self, request, *args, **kwargs):
...
     model = Volunteer
     form_class = ReportForm
     template_name = "volunteer/report.html"
-
+    
+    def get(self, request, *args, **kwargs):
+        volunteer_id = self.kwargs['volunteer_id']
+        event_list = get_signed_up_events_for_volunteer(volunteer_id)
+        return render(request, 'volunteer/report.html',
+                      {'event_list': event_list})
```
Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: Great!

File: vms/volunteer/views.py (hunk 1 above)
Comment (kamsuri) [reply vms/volunteer/views.py]: Fixed this issue and all the spacing problems.

File: vms/volunteer/views.py (hunk 2 above)
Comment (kamsuri) [reply vms/volunteer/views.py]: Thanks.

**Comment by arunikayadav42** [comment]
Can anybody please tell me if anybody is working on this issue?If not i would like to get started please,please help me out

**Comment by arunikayadav42** [comment]
I want to get started with this issue .Can somebody please help me out?

**Pull request merged**